Source/WebCore/Modules/WebGPU/GPURenderPassEncoder.cpp

```diff
@@ -147,13 +147,11 @@ void GPURenderPassEncoder::endOcclusionQuery()
 
 void GPURenderPassEncoder::executeBundles(Vector<RefPtr<GPURenderBundle>>&& bundles)
 {
-    Vector<std::reference_wrapper<WebGPU::RenderBundle>> result;
-    result.reserveInitialCapacity(bundles.size());
-    for (const auto& bundle : bundles) {
-        if (!bundle)
-            continue;
-        result.uncheckedAppend(bundle->backing());
-    }
+    auto result = WTF::compactMap(bundles, [](auto& bundle) -> std::optional<std::reference_wrapper<WebGPU::RenderBundle>> {
```

Comment:
I had to stop and remind myself that `map` isn't just a data type, it's an algorithm :-D